samples/outbox/multi-tenant/Version_6/Sender/app.config

```diff
@@ -19,4 +19,12 @@
       </providers>
     </roleManager>
   </system.web>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="NServiceBus.Core" publicKeyToken="9fc386479f8a226c" culture="neutral" />
```

Comment:
yeah i have started doing many of my nuget updates using sublime text search and replace